### PR TITLE
Updated Sample application

### PR DESCRIPTION
**Pull request by sjoerd0301**

### CODE REVIEW

File: ExoPlayer.Core/Additions/DownloadService.cs [hunk 1]
```diff
@@ -0,0 +1,577 @@
...
+        private string channelId;
+        private int channelName;
+
+        private DownloadManager downloadManager;
+        private DownloadManagerListener downloadManagerListener;
+        private int lastStartId;
+        private bool startedInForeground;
+
+        ///HACK Needed to instantiate the service, Java calls super() from method body, C# for some reason can't do ctor : base(args) or ctor(args) -> ctor() : this(args)
+        ///TODO Must call Initialize() from child class.
```
Comment (martijn00) [inline ExoPlayer.Core/Additions/DownloadService.cs]: Need to fix the TODO?

File: ExoPlayer.Core/ExoPlayer.Core.csproj [hunk 2]
```diff
@@ -7,6 +7,7 @@
     <PackageId>Xam.Plugins.Android.ExoPlayer.Core</PackageId>
     <IsBindingProject>true</IsBindingProject>
     <EnableDefaultItems>false</EnableDefaultItems>
+    <!--<AndroidClassParser>class-parse</AndroidClassParser>-->
```
Comment (martijn00) [inline ExoPlayer.Core/ExoPlayer.Core.csproj]: Why is this changed?

File: ExoPlayer.Core/ExoPlayer.Core.csproj [hunk 3]
```diff
@@ -18,5 +19,9 @@
     <AndroidResource Include="Resources\**\*.xml" SubType="Designer" Generator="MSBuild:UpdateAndroidResources" />
   </ItemGroup>
   
+  <ItemGroup>
```
Comment (martijn00) [inline ExoPlayer.Core/ExoPlayer.Core.csproj]: Remove this section

File: Samples/Main/DownloadService.cs [hunk 4]
```diff
@@ -0,0 +1,578 @@
...
+using Com.Google.Android.Exoplayer2.Util;
+using Java.IO;
+using Utils = Com.Google.Android.Exoplayer2.Util.Util;
+using Com.Google.Android.Exoplayer2.Scheduler;
+using static Com.Google.Android.Exoplayer2.Offline.DownloadManager;
+using Java.Lang;
+
+namespace Com.Google.Android.Exoplayer2.Demo
+{
+    public abstract class DownloadService : Service
```
Comment (martijn00) [inline Samples/Main/DownloadService.cs]: Class needs to be removed since it is in Core.

**Comment by martijn00** [comment]
@sjoerd0301 Can you make sure to check white space or double empty lines, and clean up the overall code style?

**Comment by martijn00** [comment]
@MKuckert or @bspinner can you guys take a look at this and review it?

File: ExoPlayer.Core/Additions/DownloadService.cs (hunk 1 above)
Comment (sjoerd0301) [reply ExoPlayer.Core/Additions/DownloadService.cs]: Initialize needs to be called in the implementing class.

File: ExoPlayer.Core/ExoPlayer.Core.csproj (hunk 3 above)
Comment (sjoerd0301) [reply ExoPlayer.Core/ExoPlayer.Core.csproj]: Fixed

File: ExoPlayer.Core/ExoPlayer.Core.csproj (hunk 2 above)
Comment (martijn00) [reply ExoPlayer.Core/ExoPlayer.Core.csproj]: I think the SDK extras already set this by default. Please remove the whole line.
